src/org/zaproxy/zap/extension/ascanrulesBeta/ZapAddOn.xml

```diff
@@ -15,6 +15,7 @@
 	</extensions>
 	<ascanrules>
 		<ascanrule>org.zaproxy.zap.extension.ascanrulesBeta.BackupFileDisclosure</ascanrule>
+		<ascanrule>org.zaproxy.zap.extension.ascanrulesBeta.BufferOverflow</ascanrule>
```

Comment:
The _changes_ should be updated to indicate the promotion of Buffer Overflow to beta.